src/App.js

```diff
@@ -41,6 +42,7 @@ function App() {
           <Route path="/reservations" element={<Navbar Page={Reservations} />} />
           <Route path="/myReservations" element={<Navbar Page={MyReservations} />} />
           <Route path="/addCar" element={<Navbar Page={AddNewCar} />} />
+          <Route path="/detail/cars/delete" element={<DeletePage />} />
```

Comment:
- Since there isn't a link to the deletion page I supposed you get there with this path ☝🏼. Unfortunately when I type this path in the browser I get the following error.
![error](https://user-images.githubusercontent.com/38721698/143269252-528e5221-fb19-4a17-8f1e-8d4a5122d50c.png)